### PR TITLE
Add testimonial for Sean Shüping

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Add testimonial for Sean Shüping

## QA

- Load feature branch
- go to http://0.0.0.0:8041/
- Scroll down to testimonials
- Verify that Sean Shüping and Tom Barber testimonials load randomly on a 50/50 split

Fixes https://github.com/canonical-web-and-design/juju.is/issues/14

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -254,21 +254,43 @@ <h4>JAAS is Juju as a service.</h4>
...
+          <blockquote class="p-pull-quote u-no-margin--bottom">
+            <p class="p-pull-quote__quote u-no-margin--bottom">
+              For me, speed and simplicity are paramount. It may sound silly, but
+              a lot of the stuff I do in my lab with Juju is out of pure laziness.
+              Juju frees my brain to focus on other things. While Juju itself is
+              really intuitive and easy to use, I was pleasantly surprised by how
+              welcoming and humble the Juju community is. The community is a big
+              reason why I’ve stuck with it.
+            </p>
+            <cite class="p-pull-quote__citation">Sean Shüping, <a href="https://devzero.co.za/">/dev/zero</a></cite>
```
Comment (anthonydillon) [inline templates/index.html]: External link?

**Comment by barrymcgee** [comment]
@anthonydillon Two is not enough to justify a carousel or imo, not enough to list stacked either which is why I alternated them. I guess it's really a design call, @spencerbygraves?

**Comment by hatched** [comment]
@barrymcgee my vote would be to convert the section to a 2 column area and put one on each side.

**Comment by anthonydillon** [comment]
Much nicer :+1:

**Comment by timClicks** [comment]
Thanks for adding this.

**Comment by barrymcgee** [comment]
@timClicks No problem :)

**Pull request merged**